### PR TITLE
#184361232 Feat:admin can get all orders

### PR DESCRIPTION
**Pull request by emilenfc**
# Description

In this feature, I added the root that allow admin to get all orders made by a buyer . also added order to display paymant status(which will be activated well when method of payment finished)
I also modify the way to get data in order so that the order will contain the detail of product and quantity

# Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have created a loom video for the new feature or enhancement
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ISHIMWE-Aime — APPROVED** [review]
LGTM

File: src/routes/order.routes.ts [hunk 1]
```diff
@@ -16,4 +17,5 @@ orderRouter.get("/all", protectRoute, ViewOrders); // view all orders
 orderRouter.delete("/alll", protectRoute, clearOrders); // delete all orders
```
Comment (shyakadavis) [inline src/routes/order.routes.ts:17]:
```suggestion
orderRouter.delete("/all", protectRoute, clearOrders);
```

**Review by SergeInGithub — COMMENTED** [review]
this is good. Kindly address the changes @shyakadavis requested and everything will be great

File: src/routes/order.routes.ts (hunk 1 above)
Comment (shyakadavis) [reply src/routes/order.routes.ts:17]: Remember to update the docs as well to reflect this. 🥂

**Review by shyakadavis — APPROVED** [review]
This is fantastic.
💪🏽 

LGTM 🚀

**Review by SergeInGithub — DISMISSED** [review]
LGTM

**Review by mucyoalexaxel — APPROVED** [review]
LGTM

**Pull request merged**